templates/windows/chocolateyInstall.ps1.erb

```diff
@@ -0,0 +1,5 @@
+<%- dirnames = get_directories.map {|d| d.path } -%>
+<%- dirnames.each do |dir| -%>
+  <%- windows_path = platform.convert_to_windows_path(dir) -%>
+Copy-Item <%= "#{windows_path.sub("C:", "C:/ProgramData/chocolatey/lib/#{project.name}")}" %> <%= windows_path %> -recurse
```

Comment:
Copy-Item is fine. You don't need to go further than one call though (but seeing an example of this in action would be helpful) because of -recurse.

Note that you will also need -force to overwrite.

~~~powershell
$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
$sourceDir = Join-Path $toolsDir 'pl-build-tools'
$destination = 'C:\tools\pl-build-tools'
# the \* is very important
Copy-Item $sourceDir\* $destination -Recurse -Force
~~~